src/Symfony/Component/Validator/Tests/Constraints/UniqueTest.php

```diff
@@ -35,6 +36,20 @@ public function testAttributes()
         self::assertSame(['my_group'], $cConstraint->groups);
         self::assertSame('some attached data', $cConstraint->payload);
     }
+
+    public function testInvalidNormalizerThrowsException()
```

Comment:
Right now, your new tests are only executed on PHP 8 because the whole class is annotated as `@requires PHP 8`. We should change that and move that annotation down to the `testAttributes` method.